libs/blocks/quiz-results/quiz-results.css

```diff
@@ -7,3 +7,59 @@
 .quiz-results.show {
   visibility: unset;
 }
+
+.section.phone-only,
```

Comment:
These generic ".section" selectors will likely not be acceptable to Milo core. They begrudgingly accepted my addition of the section class to quiz-results as a necessity, but they may draw the line at us adding css that can affect other sections in a page. Maybe we need to scope these rules to .quiz-results?

Reply:
Rubber-stamping Cody's suggestion, I think scoping them to .quiz-results should do the trick. 

.quiz-results .section.phone-only …

Reply:
Just to be clear here, you can just swap the .section selectors with .quiz-results, since .section is added manually in the JS to quiz-results anyway. Except in instances in which you are trying to target sections inside of fragments loaded by quiz-results - though that situation could potentially be very confusing for authors.

Reply:
Hi @fullcolorcoder, I think you're correct about it being beneficial to have this sort of thing available in Milo in a generic fashion for handling breakpoint specific content. But, when I've discussed the idea earlier, the reply was that Milo wasn't going there yet. For now, at least, I think we should finish this PR so we're done, and afterwards, raise another milo-community thread to ask about moving it so it's generic.